src/main/java/at/hannibal2/skyhanni/features/garden/AnitaMedalProfit.kt

```diff
@@ -87,7 +97,14 @@ class AnitaMedalProfit {
 
         val (name, amount) = ItemUtils.readItemAmount(itemName) ?: return
 
-        var internalName = NEUItems.getInternalNameOrNull(name)
+        var internalName: NEUInternalName? = null
+        for ((wrongName, correctName) in farmingToolIDs) {
+            if (wrongName == NEUItems.getInternalNameOrNull(name)) internalName = correctName
+        }
+        for ((wrongName, correctName) in farmingToolNames) {
+            if (itemName == wrongName) itemName = correctName
+        }
```

Comment:
Instead of having a second loop here, in the previous loop when it is found also get the display name with `correctName.itemName` this ensures that if hypixel changes some display format in the future none of this breaks